src/time/time_format/formatter.cr

```diff
@@ -176,6 +176,101 @@ struct TimeFormat
       {negative, hours, minutes}
     end
 
+    # Compute week of the year, monday is the first day of the week
+    def week_of_year_monday_1_7
+      io << Formatter.iso8601_week_of_year(time, 1)
+    end
+
+    # Compute week of the year, sunday is the first day of the week
+    def week_of_year_sunday_0_6
+      io << Formatter.iso8601_week_of_year(time, 0)
+    end
+
+    # Internal helper wethod: Figure how many weeks into the year
+    def self.week_of_year(time, firstweekday)
```

Comment:
Nope doesn't work. Your example doesn't match since the caller is non-static.

That's a better example and the error I get:
http://carc.in/#/r/djl